.github/workflows/deploy-docker.yaml

```diff
@@ -0,0 +1,88 @@
+name: Deploy container to DockerHub
+
+on:
+  workflow_dispatch:
+
+jobs:
+ docker-deploy:
+    name: Docker Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Maximize build space
```

Comment:
do we need this?

Reply:
Not for this specific container, but if we have to build something larger its nice to have the larger space.